src/main/java/org/example/XMLValidator.java

```diff
@@ -0,0 +1,175 @@
+package org.example;
+
+import org.example.model.User;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+
+import javax.xml.XMLConstants;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.Source;
+import javax.xml.transform.stream.StreamSource;
+import javax.xml.validation.Schema;
+import javax.xml.validation.SchemaFactory;
+import javax.xml.validation.Validator;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class XMLValidator {
+    public static void main(String[] args) throws SAXException, IOException, ParserConfigurationException {
+
+        File userSchemaFile = new File("src/main/resources/User.xsd");
```

Comment:
You could put all the xmls into one file and all the xsds into one file so you don't need to create this many File objects

Reply:
Will do. Interpreted his instructions as one pair of files for at least 5 classes